validate-tvdb-seasons.py

```diff
@@ -1,24 +1,25 @@
 import os, sys, subprocess
 import tvdb_v4_official, yaml
 from dotenv import load_dotenv
-from deepdiff import DeepDiff, Delta
+from deepdiff import DeepDiff
+import urllib
 
 
 def getTvdbId(showName):
     # Get TVDB ID of show
     showId = None
     series = None
-    searchResults = tvdb.search(showName)
+    searchResults = tvdb.search(showName, type="series", country="jpn")
```

Comment:
Don't forget that an anime in Anilist terms can be originated from: Japan, South Korea, China, and Taiwan